deerhunt/src/components/Countdown.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from "react";
+import { Text } from "@chakra-ui/react";
+
+class CountDownTimer extends Component {
```

Comment:
DAANG you put that work in lol

Reply:
I borrowed this code...no one is writing their own countdown component 😉